Text/CustomEntitySearch/CustomEntitySearch.cs

```diff
@@ -46,12 +46,17 @@ public static class CustomEntitySearch
         /// 2. Words can contain special characters and numbers
         /// 3. The provided entities are not case sensitive
         /// </summary>
-
-        [FunctionName("custom-search")]
+        [FunctionName("custom-entity-search")]
         public static async Task<IActionResult> RunCustomEntitySearch(
             [HttpTrigger(AuthorizationLevel.Function, "post", Route = null)] HttpRequest req,
-            ILogger log, ExecutionContext executionContext)
+            ILogger log,
+            ExecutionContext executionContext)
         {
+            if (preLoadedWords == null)
+            {
+                preLoadedWords = WordLinker.WordLink(executionContext.FunctionAppDirectory, "json").Words;
```

Comment:
So you're also switching the default to json.

Reply:
oops, no good catch